awx/main/management/commands/replay_job_events.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015 Ansible, Inc.
+# All Rights Reserved
+
+import sys
+from optparse import make_option
+
+from django.core.management.base import NoArgsCommand
+
+from awx.main.models import (
+    UnifiedJob,
+    Job,
+    AdHocCommand,
+)
+from awx.main.consumers import emit_channel_notification
+from awx.api.serializers import (
+    JobEventWebSocketSerializer,
+    AdHocCommandEventWebSocketSerializer,
+)
+
+
+class Command(NoArgsCommand):
+
+    help = 'Replay job events over websockets'
+
+    option_list = NoArgsCommand.option_list + (
+        make_option('--job_id', dest='job_id', type='int', metavar='j',
+                    help='Id of the job to replay (job or adhoc)'),
```

Comment:
A general --help summary would be really nice, just to say what it means to "replay" a job.

Reply:
```
(awx) [root@awx awx_devel]# awx-manage replay_job_events --help
Usage: /usr/local/bin/awx-manage replay_job_events [options]

Replay job events over websockets

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  -v VERBOSITY, --verbosity=VERBOSITY
                        Verbosity level; 0=minimal output, 1=normal output,
                        2=verbose output, 3=very verbose output
  --settings=SETTINGS   The Python path to a settings module, e.g.
                        "myproject.settings.main". If this isn't provided, the
                        DJANGO_SETTINGS_MODULE environment variable will be
                        used.
  --pythonpath=PYTHONPATH
                        A directory to add to the Python path, e.g.
                        "/home/djangoprojects/myproject".
  --traceback           Raise on CommandError exceptions
  --no-color            Don't colorize the command output.
  --job_id=j            Id of the job to replay (job or adhoc)
```